plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveTableHandle.java

```diff
@@ -427,6 +318,11 @@ public Optional<Long> getMaxScannedFileSize()
         return maxScannedFileSize;
     }
 
+    public TableBuilder toBuilder()
```

Comment:
Instead of this we should have `public static Builder buildFrom(HiveTableHandle table)`